Update `CHANGELOG.md` for the 0.3.1 release (#12)
CoreOffice/OLEKit: CHANGELOG.md

# 0.3.1 (25 January 2021)

> This release fixes issue that it can't get file size from attributes in some cases (such as iOS Document-based App with SwiftUI)
> during creating `OLEFile` instances.

Reviewer: ```suggestion
This release fixes an issue with file size attributes in some cases (such as iOS Document-based App with SwiftUI)
during `OLEFile` creation.
```
